packages/app-store/googlecalendar/lib/CalendarService.ts

```diff
@@ -151,14 +153,18 @@ export default class GoogleCalendarService implements Calendar {
       },
       updateTokenObject: async (token) => {
         this.myGoogleAuth.setCredentials(token);
-        await prisma.credential.update({
+
+        const { key } = await prisma.credential.update({
           where: {
             id: credential.id,
           },
           data: {
             key: token,
           },
         });
+
+        // Update cached credential as well
+        this.credential.key = key;
```

Comment:
This is important line which fixes the issue.

We update the token in DB. But in case of google-calendar there is a cached value here as well. We should keep it upto-date in case someone use this value.